CONTRIBUTING.md

```diff
@@ -94,75 +94,73 @@ A commit message must start with the corresponding ticket number in GitHub (#TIC
 In general the HL7 [FHIR naming conventions](http://wiki.hl7.org/index.php?title=FHIR_Guide_to_Designing_Resources#Naming_Rules_.26_Guidelines) apply. Essentially these conventions ask for **consistency** and **precision** (i.e. minimizing ambiguity, while ensuring the meaning is easily understood) when naming fields, resources or operations.
 
 Most of these guidelines are suggestions, except the following rules that *must* be followed:
--  be lowerCamelCase for elements, UpperCamelCase for resources, be lowercase for operations
+-  be lowerCamelCase for elements, lower case for resources and for operations
```

Comment:
This is slightly confusing (and was already).
We should add examples, because it is unclear that "countryCodes" is still valid camelCase (because it is an element) 

And possibly also Split this into three listings, as elements, resources and operations are very different things

Reply:
👍 I agree examples should be added here to clarify.

Reply:
Changes made together in TC FHIR Call on 17/6/2021